docs/superpowers/plans/2026-04-28-easy-policy-interpretation.md

```
@@ -0,0 +1,2286 @@
+
+---
+
+## Task 15: ENTITIES.md 갱신
+
+**Files:**
+- Modify: `docs/ENTITIES.md`
+
+- [ ] **Step 1: 3.1 Guide 섹션 교체**
+
+`docs/ENTITIES.md`의 다음 부분:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)